src/features/subdomain-view/subdomain-table/SubdomainTable/SubdomainTable.tsx

```diff
@@ -0,0 +1,104 @@
+//- React Imports
```

Comment:
Makes sense ! I've moved the files around a lot since this review with the intention of following the practices you've outlined. Please let me know if the structure still needs tweaking 🙂

Reply:
<img width="332" alt="Screenshot 2022-09-05 at 14 26 29" src="https://user-images.githubusercontent.com/39112648/188461474-5812ea7e-4035-4a93-a7fa-389d91d65db8.png">